src/containers/AdminCampaignEdit/sections/CampaignContactsForm/components/ConfigureColumnMappingDialog.tsx

```diff
@@ -99,15 +99,15 @@ const ConfigureColumnMappingDialog: React.FC<ConfigureColumnMappingDialogProps>
   };
 
   useEffect(() => {
-    if (open && !parseComplete) {
+    if (open && !parseComplete && contactsFile) {
```

Comment:
Did you mean `!!contactsFile` to specify this as a boolean? Type is of `File | null`

Reply:
Yeah do you feel strongly about specifying the boolean conversion vs. the current truthy check?

Reply:
I think it's a good practice to get into but not absolutely necessary. Making sure you're intentional about typing rather than letting javascript convert things for you is a really good mindset to have and it often prevents bugs.

Reply:
sorry probably should have said "nice to have". I'm not really at my best haha

Reply:
Got you, how do you feel about a `contactsFile != null` instead? My 🧠 is forever going to do a double take when it sees `!!`s

Reply:
I'd prefer you leave as is rather than using a single exclamation point 😅 

sorry for the trouble